### ISSUE TITLE
[BITV]: Rework native tooltips on NcAvatar

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/nextcloud-vue#3677]
## Problem

To show a tooltip on a several components we currently use v-tooltip. An example can be found in https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcAvatar/NcAvatar.vue#L112

v-tooltip is not accessible and we want to migrate to native tooltips (title attribute) instead. An overview can be found in https://github.com/nextcloud/nextcloud-vue/issues/2503.

## Solution / ToDo:

- [ ]  Add a `title` prop to NcAvatar component that will render as `title` attribute on the HTML element. Remove custom tooltips.
- [ ] Show valid value for `aria-label` every time.
